api/src/main/java/org/pmiops/workbench/firecloud/FireCloudServiceImpl.java

```diff
@@ -55,12 +61,15 @@
   private final Provider<StaticNotebooksApi> staticNotebooksApiProvider;
   private final FirecloudRetryHandler retryHandler;
   private final Provider<GoogleCredential> fcAdminCredsProvider;
+  private final Provider<CloudStorageService> cloudStorageServiceProvider;
   private final ServiceAccounts serviceAccounts;
+  private final LoadingCache<String, GoogleCredential> garbageCollectionSACredentials;
```

Comment:
In Perf we may need to get these credentials hundreds of times per run.  Let's not hit GCS and auth APIs every time.